hooks/consul.py

```diff
@@ -71,15 +71,14 @@ def install_web_ui(version, destination_directory='/usr/share/consul'):
                           hash_type='sha256')
 
 
-def find_sha256sum(file_name):
+def find_sha256sum(version, file_name):
     ''' Find the expected checksum in the SHA256SUMS file by file name.'''
-    # Get the version from the target file name.
-    version = file_name.split('_')[0]
     # The SHA256SUMS files have a version prefix.
-    shasum_file_name = '{0}_SHA256SUMS'.format(version)
-    shasum_file_url = URL_PREFIX + shasum_file_name
+    shasum_file_name = 'consul_{0}_SHA256SUMS'.format(version)
+    shasum_url = '{0}/{1}/{2}'.format(URL_PREFIX, version, shasum_file_name)
+    print('Get the checksum file {0}'.format(shasum_url))
```

Comment:
Thanks, should have used log, but this url failed in my testing and I didn't know why so I printed it while debugging and thought it was useful after debugging.